consensus/src/epoch_manager.rs

```diff
@@ -1198,3 +1201,18 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
         }
     }
 }
+
+fn new_signer_from_storage(
+    author: Author,
+    backend: &SecureBackend,
+) -> aptos_types::validator_signer::ValidatorSigner {
```

Comment:
we probably should get one Arc\<signer\> to share instead of parsing the backend multiple times